EasyPost.Tests/ReportTest.cs

```diff
@@ -41,12 +49,13 @@ private static async Task TestRetrieveReport(string cassetteName, string reportT
         {
             VCR.Replay(cassetteName);
 
-
             Report report = await CreateBasicReport(reportType);
 
             Report retrievedReport = await Report.Retrieve(report.id);
 
-            Assert.IsInstanceOfType(report, typeof(Report));
+            Assert.IsInstanceOfType(retrievedReport, typeof(Report));
+            // Some reports take a long time to generate, so the URL won't be available immediately
+            // Assert.IsNotNull(retrievedReport.url);
```

Comment:
We don't test against the URL, remove it completely and have this match the other libs.